### ISSUE TITLE
E2E k8s tests failing

### ISSUE THREAD

**Issue by GabyCT** [kata-containers/tests#3893]
It seems that these jobs have been failing for a long time
http://jenkins.katacontainers.io/job/kata-containers-2.0-clh-containerd-k8s-conformance-e2e/
http://jenkins.katacontainers.io/job/kata-containers-2.0-containerd-k8s-conformance-e2e/
http://jenkins.katacontainers.io/job/kata-containers-2.0-tests-fedora-PR-crio-k8s-e2e-complete/

/cc @jcvenegas @chavafg @fidencio @fgiudici 

Should we remove these jobs? Are they really important?

**Comment by wainersm** [comment]
Hi @fidencio @GabyCT ,

First of all, it is likely those jobs are producing bogus results because there is a bug on the kata-webhook; the tests are actually checking runc rather than kata containers. See the problem description in https://github.com/kata-containers/tests/issues/3639 and the  proof it is not working in https://github.com/kata-containers/tests/pull/3647 . This alone is enough for us to  disable all those jobs.

The jobs *kata-containers-2.0-clh-containerd-k8s-conformance-e2e* and *kata-containers-2.0-containerd-k8s-conformance-e2e* don' t have coverage on the job I maintain on OpenShift CI. If we decide to keep them running, it should be good to find owners to maintain them.

Regarding the *kata-containers-2.0-tests-fedora-PR-crio-k8s-e2e-complete* job, there might be some overlap with the job on OpenShift CI. I tend to say "let' s disable it" but I prefer to investigate the gaps/overlaps before taking any decision. 
 
 In summary, IMHO we should:

 1) Disable all those jobs
 2) Fix https://github.com/kata-containers/tests/issues/3639
 3) Find owners for each of those
 4) Re-enable them (the owners should decide that actually)

What do you think?

**Comment by GabyCT** [comment]
thanks @wainersm , wdyt @chavafg @jcvenegas ?

**Comment by jcvenegas** [comment]
@GabyCT I have disabled the cron job from  `kata-containers-2.0-clh-containerd-k8s-conformance-e2e`

**Comment by GabyCT** [comment]
@jcvenegas I still see it enabled  http://jenkins.katacontainers.io/job/kata-containers-2.0-clh-containerd-k8s-conformance-e2e/

**Comment by GabyCT** [comment]
I will disable http://jenkins.katacontainers.io/job/kata-containers-2.0-containerd-k8s-conformance-e2e/ 
^ / cc @chavafg

**Comment by jcvenegas** [comment]
I changed to name to  give more context, http://jenkins.katacontainers.io/view/Cloud-Hypervisor%20Jobs/job/kata-containers-2.0-ubuntu-main-branch-x86_64-cloud-hypervisor-k8s-containerd-full-tests-repo/
